File: ChatAppBackend/src/main/java/com/gapshap/app/payload/TypeStatusResponse.java
package com.gapshap.app.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TypeStatusResponse {

	private Long id;
	private String userName;
	private String profileName;
	private String email;
	private String toEmail;
	private String conversationId;
	private String userStatus;
}

File: ChatAppAngular/src/app/user/payload/type-status-response.ts
export class TypeStatusResponse {
    id: number = 0;
    userName: string = ' ';
    profileName: string = ' ';
    email: string = ' ';
    toEmail = " ";
    conversationId: string  = ' ';
    userStatus: string = 'ONLINE';
    constructor(id: number, name: string, email: string, conversationId: string,userStatus:string,toEmail:string,profileName:string) {
        this.id = id;
        this.userName = name;
        this.email = email;
        this.toEmail = toEmail;
        this.conversationId = conversationId;
        this.userStatus = userStatus;
        this.profileName=profileName;
    }
}
